code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -702,12 +703,163 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
+		W.sight_range *= 3
+	for(var/obj/item/clockwork/clockwork_proselytizer/P in all_clockwork_objects) //Proselytizers no longer require alloy
+		P.uses_alloy = FALSE
+	for(var/obj/item/clockwork/tinkerers_daemon/D in all_clockwork_objects) //Daemons produce components twice as quickly
+		D.production_interval /= 2
+	for(var/obj/structure/clockwork/mending_motor/M in all_clockwork_objects) //Mending motors no longer require alloy
+		M.uses_alloy = FALSE
+	spawn(600)
+		for(var/obj/structure/clockwork/ocular_warden/W in all_clockwork_objects)
+			W.damage_per_tick = initial(W.damage_per_tick)
+			W.sight_range = initial(W.sight_range)
+		for(var/obj/item/clockwork/clockwork_proselytizer/P in all_clockwork_objects)
+			P.uses_alloy = TRUE
+		for(var/obj/item/clockwork/tinkerers_daemon/D in all_clockwork_objects)
+			D.production_interval = initial(D.production_interval)
+		for(var/obj/structure/clockwork/mending_motor/M in all_clockwork_objects)
+			M.uses_alloy = TRUE
+	spawn(3000) //5 minutes
+		clockwork_generals_invoked["nezbere"] = FALSE
+	qdel(src)
+	return 1
+
+/datum/clockwork_scripture/invoke_sevtug //Invoke Sevtug, the Formless Pariah: Allows the invoker to silently control the mind of a defined target for one minute.
+	name = "Invoke Sevtug, the Formless Pariah"
+	desc = "Taps the limitless power of Sevtug, one of Ratvar's four generals. The mental manipulation ability of the Pariah allows its wielder to silently dominate the mind of a defined target \
+	for one minute. <b>Note that this process is very delicate and very many things may prevent you from ever returning to your old form!</b>"
+	invocations = list("V pnyy hcba lbh, Sevtug!!", "Yrg lbhe cbjre fung-gre gur fnavgl bs gur jrnx-zvaqrq!!", "Yrg lbhe graqevyf ubyq fjnl bire nyy!!")
+	channel_time = 150
+	required_components = list("belligerent_eye" = 3, "vanguard_cogwheel" = 3, "guvax_capacitor" = 6, "hierophant_ansible" = 3)
+	tier = SCRIPTURE_REVENANT
+	var/mind_control_target
+
+/datum/clockwork_scripture/invoke_sevtug/can_recite()
+	if(!..())
+		return 0
+	if(ratvar_awakens)
+		invoker << "<font color='#AF0AAF'><b><i>\"Qb lbh ernyyl guvax nalguvat v pna qb evtug abj jvyy pbzcner gb Ratvar's cbjre?.\"</b></i></font>\n\
+		<span class='warning'>Sevtug will not grant his power while Ratvar's dwarfs his own!</span>"
+		return 0
+	if(clockwork_generals_invoked["sevtug"])
+		invoker << "<font color='#AF0AAF'><b><i>\"Vf vg ernyyl fb uneq - rira sbe n fvzcyrgba yvxr lbh - gb tenfc gur pbaprcg bs jnvgvat?\"</b></i></font>\n\
+		<span class='warning'>Sevtug has already been invoked recently! You must wait several minutes before calling upon the Formless Pariah.</span>"
+		return 0
+	while(!mind_control_target)
+		var/potential_target = stripped_input(invoker, "Enter the actual name of a target to dominate (case-sensitive).", name)
+		if(!potential_target)
+			return 0
+		for(var/mob/living/L in living_mob_list)
+			if(L.real_name == potential_target)
+				if(is_clockwork_cultist(L))
```

Comment:
I want to keep it in so players don't get confused as to why the name they enter isn't working, but I ca n remove it if it truly bothers you.